components/AppHeader.vue

```diff
@@ -1,16 +1,28 @@
 <template>
   <CHeader class="header-sticky p-0" :class="{ 'shadow-sm': isScrolled }">
-    <CContainer id="headerNavContainer" fluid class="border-bottom mt-1 justify-content-start">
-      <CHeaderToggler class="d-lg-none" @click="toggleSidebarVisibility">
-        <span data-testid="toggle-sidebar-button">
-          <CIcon icon="cilMenu" size="lg" />
-        </span>
-      </CHeaderToggler>
-      <div class="header-brand px-2">
-        <NuxtLink prefetch-on="interaction" to="/" class="nav-link">
-          <CIcon icon="cilGlobeAlt" size="lg" />
-          <span id="appTitle">DAEDALUS Explore</span>
-        </NuxtLink>
+    <CContainer id="headerNavContainer" fluid class="border-bottom mt-1 justify-content-between">
+      <div class="d-flex">
+        <CHeaderToggler class="d-lg-none" @click="$emit('toggleSidebarVisibility')">
+          <span data-testid="toggle-sidebar-button">
+            <CIcon icon="cilMenu" size="lg" />
+          </span>
+        </CHeaderToggler>
+        <div class="header-brand px-2">
+          <NuxtLink prefetch-on="interaction" to="/" class="nav-link">
+            <CIcon icon="cilGlobeAlt" size="lg" />
+            <span id="appTitle">DAEDALUS Explore</span>
+          </NuxtLink>
+        </div>
+      </div>
+      <div class="d-none d-sm-block">
+        <img
+          data-testid="ji-logo-header"
+          style="width: 150px;"
+          class="img-fluid"
+          :title="versionTooltipContent"
```

Comment:
Ticket to move the version info into the About page - putting it in the tooltip was always just a placeholdhttps://mrc-ide.myjetbrains.com/youtrack/issue/JIDEA-178/Move-version-information-into-the-About-pageer!